### PR TITLE
Add Explore Capabilities and Learning Resources widgets

### PR DESCRIPTION
**Pull request by florkbr**
See https://github.com/RedHatInsights/learning-resources/pull/45 & https://github.com/RedHatInsights/learning-resources/pull/46

and

https://github.com/RedHatInsights/landing-page-frontend/pull/545

### CODE REVIEW

File: rest/service/dashboardTemplate.go [hunk 1]
```diff
@@ -20,6 +20,14 @@ var (
 			Scope:  "notifications",
 			Module: "./DashboardWidget",
 		},
+		models.LearningResources: models.ModuleFederationMetadata{
+			Scope:  "learningResources",
+			Module: "./BookmarkedLearningResourcesWidget",
```
Comment (florkbr) [inline rest/service/dashboardTemplate.go:25]: @karelhala FYI I fixed the typo in this PR. Let me know if you change the name to something else.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by Hyperkid123** [comment]
@florkbr I'll wait for Karel's response.

**Comment by florkbr** [comment]
Also added widgets from: https://github.com/RedHatInsights/landing-page-frontend/pull/548 (simple services widgets)

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**